app/views/candidate_interface/sign_up/new.html.erb

```diff
@@ -29,7 +29,7 @@
           <li>checking you’re safe to work with children</li>
         </ul>
 
-        <%= f.govuk_check_box :accept_ts_and_cs, true, multiple: false, link_errors: true, label: { text: t('authentication.sign_up.accept_terms_checkbox') } %>
+        <%= f.govuk_check_box :accept_ts_and_cs, @sign_up_form.accept_ts_and_cs, multiple: false, link_errors: true, label: { text: t('authentication.sign_up.accept_terms_checkbox') } %>
```

Comment:
This solves the bug, but I'm not convinced it is the correctly solution. I've had a look through the (docs)[https://www.rubydoc.info/gems/govuk_design_system_formbuilder/GOVUKDesignSystemFormBuilder/Builder#govuk_check_box-instance_method] which suggests that this is the attribute for controlling the checkbox value. 

But having a look at other checkboxes in the codebase, we haven't implemented similar changes elsewhere. The second attribute is left as `true` rather than taking in the property on the object.

Reply:
`'true'` seems to work just as well

Reply:
`'true'` does work on the page but I ran in to some difficulty when writing the tests.

Both `true` and `'true'` add a `value='true'` attribute to the input element in both the unchecked and checked states, which meant that you couldn't distinguish between the states for the test. I don't really understand how it is working on the page.

`@sign_up_form.accept_ts_and_cs` only adds `value='on'` in the checked state, which is means we can test it easily and it is easier to reason about how it is working.

Reply:
Took me a bit to understand this! 

For my understanding: what the form builder does is compare `@sign_up_form.accept_ts_and_cs` with whatever is the second argument to `govuk_check_box`.

- When we use the boolean `true`, the boolean is cast to the string `"true"` as the `value=`. This means that the for compares `true == "true"`
- When we use `"true"` or _any_ other string, it will work because we compare the the correct things
- When we use  `@sign_up_form.accept_ts_and_cs`, this is initially nil, so no `value=` is provided. When submitting we then submit the string "`on`", which is compared correctly again